models/application.js

```diff
@@ -1,53 +1,31 @@
 'use strict';
-const { sumBy } = require('lodash');
-const moment = require('moment');
-
 module.exports = function(sequelize, DataTypes) {
-    return sequelize.define(
-        'application',
-        {
-            id: {
-                primaryKey: true,
-                type: DataTypes.UUID,
-                defaultValue: DataTypes.UUIDV4,
-                allowNull: false
-            },
-            user_id: {
-                type: DataTypes.INTEGER,
-                references: {
-                    model: 'users',
-                    key: 'id'
-                }
-            },
-            form_id: {
-                type: DataTypes.STRING,
-                allowNull: false
-            },
-            application_data: {
-                type: DataTypes.JSON,
-                allowNull: true
-            },
-            status: {
-                type: DataTypes.ENUM,
-                values: ['pending', 'complete'],
-                allowNull: false
-            }
+    return sequelize.define('application', {
+        id: {
+            primaryKey: true,
+            type: DataTypes.UUID,
+            defaultValue: DataTypes.UUIDV4,
+            allowNull: false
         },
-        {
-            getterMethods: {
-                grantAmount() {
```

Comment:
This now exists on the `summary` method in the form builder rather than being tied into the database mode (which doesn't know what fields are available)